docs/INSTALL.md

```diff
@@ -119,20 +118,40 @@ To configure/install the UPF in DPDK mode, the following changes are required:
      # In the order of (n-s1u n-sgi)
     -nhmacaddrs=(22:53:7a:15:58:50 22:53:7a:15:58:50)
     +nhmacaddrs=(b4:96:91:b4:47:b8 b4:96:91:b4:47:b9)
-
      # IPv4 route table entries in cidr format per port
      #
     @@ -111,7 +111,7 @@ sudo rm -rf /var/run/netns/pause
      make docker-build
-
      if [ "$mode" == 'dpdk' ]; then
     -       DEVICES=${DEVICES:-'--device=/dev/vfio/48 --device=/dev/vfio/49 --device=/dev/vfio/vfio'}
     +       DEVICES=${DEVICES:-'--device=/dev/vfio/184 --device=/dev/vfio/185 --device=/dev/vfio/vfio'}
         PRIVS='--cap-add IPC_LOCK'
-
      elif [ "$mode" == 'af_xdp' ]; then
     ```
 
+3. If testing with Intel 800 series network adapter (for example, E810 NIC) with DDP (Dynamic Device Personalization) enabled for GTPU, update the script file
+
+   ```
+   $ git diff scripts/docker_setup.sh
+   diff --git a/scripts/docker_setup.sh b/scripts/docker_setup.sh
+   index 7aff6a6..c5afb42 100755
+   --- a/scripts/docker_setup.sh
+   +++ b/scripts/docker_setup.sh
+   @@ -218,6 +218,7 @@ fi
+    docker run --name bess -td --restart unless-stopped \
+           --cpuset-cpus=12-13 \
+           --ulimit memlock=-1 -v /dev/hugepages:/dev/hugepages \
+   +       -v /lib/firmware/intel:/lib/firmware/intel \
+           -v "$PWD/conf":/opt/bess/bessctl/conf \
+           --net container:pause \
+           $PRIVS \
+   ```
+
+## Configuration: CNDP mode
+
+To configure/install the UPF in CNDP mode, please refer [CNDP_README.md](./CNDP_README.md)
+
+
```

Comment:
Please also add item in the table of contents (above):
```markdown
  * [Configuration: CNDP mode](#configuration-cndp-mode)
```

Reply:
ok